sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -150,9 +150,9 @@ This function loads data from memory to the 2d tiles/registers of Intel AMX/DPAS
 ```c++
 namespace sycl::ext::oneapi::experimental::matrix {
   template <typename Group, typename T, size_t NumRows, size_t NumCols,
-          use Use, access::address_space Space>
+          use Use, layout Layout, access::address_space Space>
```

Comment:
Why is the layout template parameter added back?
Even in the CUDA backend this is never necessary for joint_matrix_store.

Reply:
as you can see, I did not add for C on the mad function.
But for store, remember in Intel backend, you can store into A or B. So if the code has A with layout other than unused, store will not fail.